src/item_pocket.cpp

```diff
@@ -596,7 +596,9 @@ bool item_pocket::seal()
 
 void item_pocket::unseal()
 {
-    _sealed = false;
+    if( _sealed ) {
+        _sealed = false;
```

Comment:
What's the point of this change? The behaviour seems the be exactly the same as before.

Reply:
i'm not exactly married to keeping it, but my logic was that it made setting breakpoints at that part much easier if you only want to find out when it actually matters that it sets it false

Reply:
Set a conditional breakpoint then.